spock-core/src/main/java/org/spockframework/report/log/ReportLogExtension.java

```diff
@@ -1,25 +0,0 @@
-/*
- * Copyright 2013 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.spockframework.report.log;
-
-import org.spockframework.runtime.extension.AbstractGlobalExtension;
-
-public class ReportLogExtension extends AbstractGlobalExtension {
```

Comment:
The report config is actually used by other extensions, and we mainly kept this one, as you needed a global extension to initialize the config, until I changed that later on with the global ConfigObject loading.
Currently, I didn't see any need to keeps this around anymore, as `spock-reports` was removed as well.

Reply:
That was my exact question.
`spock-report` was deleted and replaced with the TODO in the extension to "integrate with JUnitPlatform ExecutionListener".
I assumed that means it is currently not working properly so was deleted but should be fixed an re-added properly later on.
So why was the whole reporting part cut out?

Besides that, there is also still docu available about now not anymore present fields in report log configuration and the `enabled` field of the extension is just a no-op, it has no effect whatsoever in Spock itself. If the reporting stays out and some 3rd party extension still uses that configuration, maybe the config object should be removed from Spock completely and that 3rd party extension nees to introduce their own config object.

Reply:
The module `spock-reports` was never released, and subsequently removed. I don't plan to add a new reporting module to the main `spock` project. There are other third-party extensions available that are maintained.

The `org.spockframework.runtime.extension.builtin.IssueExtension` still requires the `ReportLogConfiguration`. But I've removed the obsolete documentation.

Deleting this, does not mean there never will be anything new, but I think it would make sense to just add a way for extensions to publish their data via an abstracted `org.junit.platform.engine.EngineExecutionListener#reportingEntryPublished`, access to that will probably be part of #1273